11.2.основы_работы_со_списками/11.2.4.py

```diff
@@ -0,0 +1,3 @@
+primes = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71]
+
+print(primes[0:6])
```

Comment:
Если тебе нужно начать с 0-го индекса, не обязательно его указывать:
```suggestion
print(primes[:6])
```